From: Boyuan Yang <073plan@gmail.com>
Date: Sat, 27 Aug 2016 15:24:33 +0800
Subject: CMakeLists.txt: use GNUInstallDirs feature.

This feature is introduced in CMake 2.8.8, which can
greatly reduce the various problems about PATH.

Signed-off-by: Boyuan Yang <073plan@gmail.com>
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1c50834..d33fe83 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 2.8.11)
 
+include(GNUInstallDirs)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/QEverCloud/cmake/modules")
 include(QEverCloudCMakePolicies)
 SET_POLICIES()
